opensource.mk

```diff
@@ -0,0 +1,22 @@
+PROJECT = facebook/react
+LABELS = "good first issue"
```

Comment:
need to use lists here. consider case of non-uniform label delimeter from project to project